Sources/Plasma/FeatureLib/pfPasswordStore/pfPasswordStore.cpp

```diff
@@ -49,21 +49,17 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 /* Get the pfPasswordStore instance */
 pfPasswordStore* pfPasswordStore::Instance()
 {
-    static pfPasswordStore* store = nullptr;
-
-    if (store == nullptr) {
 #if defined(HS_BUILD_FOR_WIN32)
-        store = new pfWin32PasswordStore();
+    static pfWin32PasswordStore store;
 #elif defined(HS_BUILD_FOR_APPLE)
-        store = new pfApplePasswordStore();
+    static pfMacPasswordStore store;
```

Comment:
`pfMacPasswordStore` -> `pfApplePasswordStore`

This was renamed in https://github.com/H-uru/Plasma/pull/675